.github/actions/version-tag/action.yml

```
@@ -0,0 +1,41 @@
+        git config user.name "Bot"
+        git config user.email "admin@allthings.me"
+        currentVersion="$(git describe --tags `git rev-list --tags --max-count=1`)"
+        if [ "$currentVersion" = "" ]; then 
+          currentVersion="v0.0.0"
+        fi
+        latestVersion="${currentVersion//v[0-9]*.[0-9]*./}"
+        ((++latestVersion))
+        newVersion="${currentVersion%.*}.$latestVersion"
+        git tag -a $newVersion -m "Release new version: $newVersion"
+        git push origin $newVersion
+        if [ ! -z "${{ env.WORKFLOW_NAME }}" ]; then 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)